### PR TITLE
Pridec documentation in preparation for release

### PR DESCRIPTION
**Pull request by jwang125**
Comments in files and update on techrep_main.tex

### CODE REVIEW

File: src/Drivers/nlpPriDec_ex9.cpp [hunk 1]
```diff
@@ -127,7 +124,7 @@ bool PriDecMasterProblemEx9::eval_f_rterm(size_t idx, const int& n, const double
 
   ex9_recourse = new PriDecRecourseProblemEx9(nc_, nS_,S_,x,xi);
   
-  // set a few contingencies to have different sparse structure
+  // set a few contingencies to have different sparse structure to created unbalanced load
```
Comment (cnpetra) [inline src/Drivers/nlpPriDec_ex9.cpp:127]: "create"

**Review by cnpetra — APPROVED** [review]
need to give the user manual another read, otherwise lgtm

File: src/Drivers/nlpPriDec_ex8.hpp [hunk 2]
```diff
@@ -13,23 +13,30 @@
 #include <cmath>
 #include <chrono>
 
-/** This class provide an example of what a user of hiop::hiopInterfacePriDecProblem 
+/** This file provide an example of what a user of hiop::hiopInterfacePriDecProblem
```
Comment (nychiang) [inline src/Drivers/nlpPriDec_ex8.hpp:16]: provides

File: src/Drivers/nlpPriDec_ex9_sparse.cpp [hunk 3]
```diff
@@ -117,7 +116,7 @@ bool PriDecMasterProblemEx9Sparse::eval_f_rterm(size_t idx, const int& n, const
 
   ex9_recourse = new PriDecRecourseProblemEx9Sparse(nc_, nS_,S_,x,xi);
   
-  // set a few contingencies to have different sparse structure
+  // set a few contingencies to have different sparse structure to created unbalanced load
```
Comment (nychiang) [inline src/Drivers/nlpPriDec_ex9_sparse.cpp:119]: create

File: src/Drivers/nlpPriDec_ex9_sparse_raja.cpp [hunk 4]
```diff
@@ -128,12 +125,10 @@ bool PriDecMasterProblemEx9Sparse::eval_f_rterm(size_t idx, const int& n, const
   //xi[ny_-1] = double(idx/100.0+1.);
 
   PriDecRecourseProblemEx9Sparse* ex9_recourse;
-  //printf("Constructor of recourse\n");
 
   ex9_recourse = new PriDecRecourseProblemEx9Sparse(nc_, nS_, S_, x, xi, mem_space_);
   
-  //printf("Constructor of recourse complete\n");
-  // set a few contingencies to have different sparse structure
+  // set a few contingencies to have different sparse structure to created unbalanced load
```
Comment (nychiang) [inline src/Drivers/nlpPriDec_ex9_sparse_raja.cpp:131]: structures to create

File: src/Optimization/hiopAlgPrimalDecomp.hpp [hunk 5]
```diff
@@ -186,22 +207,29 @@ class hiopAlgPrimalDecomposition
...
-  private:
-    int n_;
-    double f_;
-    double* grad_;
-    double* x_;
+    private:
+      int n_;
+      double f_;
+      double* grad_;
+      double* x_;
```
Comment (nychiang) [inline src/Optimization/hiopAlgPrimalDecomp.hpp:214]: I think the old indentation was correct.

File: src/Drivers/nlpPriDec_ex9.cpp [hunk 6]
```diff
@@ -3,7 +3,7 @@
 #include "hiopAlgFilterIPM.hpp"
 
 /*
-  todo:
+  Further work:
```
Comment (nychiang) [inline src/Drivers/nlpPriDec_ex9.cpp:6]: Use "TODO" to keep consistence.

**Review by nychiang — APPROVED** [review]
I think it is better if we can cite the paper in the comment, e.g., "// this parameter is \alpha in eq(100) of the pridec paper"

**Comment by jwang125** [comment]
added one line in comments for alpha, citing the paper and referring to the user manual.

**Pull request merged**